src/System/Console/GetOpt/Generics.hs

```diff
@@ -15,8 +15,8 @@
 {-# LANGUAGE TypeSynonymInstances  #-}
 {-# LANGUAGE ViewPatterns          #-}
 
-{-# OPTIONS_GHC -fno-warn-unrecognised-pragmas #-}
 {-# OPTIONS_GHC -fno-warn-deprecated-flags #-}
+{-# OPTIONS_GHC -fno-warn-unrecognised-pragmas #-}
```

Comment:
Yeah. Given that we have travis for all supported ghc versions I wouldn't mind going that way.